ambry-store/src/main/java/com/github/ambry/store/BlobStoreStats.java

```diff
@@ -141,6 +142,16 @@ static StatsSnapshot convertStoreUsageToContainerStatsSnapshot(Map<Short, Map<Sh
       long logSegmentForecastOffsetMs, long queueProcessingPeriodInMs, long waitTimeoutInSecs,
       boolean enableBucketForLogSegmentReports, Time time, ScheduledExecutorService longLiveTaskScheduler,
       ScheduledExecutorService shortLiveTaskScheduler, DiskIOScheduler diskIOScheduler, StoreMetrics metrics) {
+    this(storeId, index, bucketCount, bucketSpanTimeInMs, logSegmentForecastOffsetMs, queueProcessingPeriodInMs,
+        waitTimeoutInSecs, enableBucketForLogSegmentReports, true, time, longLiveTaskScheduler, shortLiveTaskScheduler,
+        diskIOScheduler, metrics);
+  }
+
+  BlobStoreStats(String storeId, PersistentIndex index, int bucketCount, long bucketSpanTimeInMs,
```

Comment:
Looks `fillupLogSegmentNames` = false is only used in test? 

If test fails without your fix, and test passes with your fix. That's good enough. I don' think we need to test a wrong case by adding extra parameters.